tests/Provider/BaseProviderTest.php

```diff
@@ -33,11 +33,23 @@ public function getProvider(): MediaProviderInterface
         $adapter = $this->createMock(Adapter::class);
 
         $filesystem = $this->getMockBuilder(Filesystem::class)
-            ->onlyMethods(['get'])
             ->setConstructorArgs([$adapter])
             ->getMock();
 
-        $cdn = new Server('/uploads/media');
+        $filesystem->method('get')->willReturn(new File('my_file.txt', $filesystem));
+
+        $cdn = $this->createMock(CDNInterface::class);
```

Comment:
I don't know if we have a recent enough version of PHPUnit, but this looks like it should be a stub.

```suggestion
        $cdn = $this->createStub(CDNInterface::class);
```